site/content/en/docs/tasks/troubleshooting/troubleshooting_pods.md

```diff
@@ -0,0 +1,95 @@
+of this Pod directly.
+
+A Pod might not have the `kueue.x-k8s.io/managed` due to one of the following reasons:
+
+1. The [Pod integration is disabled](/docs/tasks/run/plain_pods/#before-you-begin).
+2. The Pod belongs to a namespace or has labels that don't satisfy the requirements of
+   the [`podOptions`](/docs/reference/kueue-config.v1beta1/#PodIntegrationOptions) configured for the Pod integration.
+3. The Pod is owned by a Job or equivalent CRD that is managed by Kueue.
+4. The Pod doesn't have a `kueue.x-k8s.io/queue-name` label and [`manageJobsWithoutQueueName`](/docs/reference/kueue-config.v1beta1/#Configuration)
+   is set to `false`.
+
+## Identifying the Workload for your Pod
+
+When using [Pod groups](/docs/tasks/run/plain_pods/#running-a-group-of-pods-to-be-admitted-together),
+the name of the Workload matches the value of the label `kueue.x-k8s.io/pod-group-name`.
+
+When using [single Pods](/docs/tasks/run/plain_pods/#running-a-single-pod-admitted-by-kueue), you can identify its corresponding
+Workload by following the guide for [Identifying the Workload of a Job](troubleshooting_jobs/#identifying-the-workload-for-your-job).
+
+## Why doesn't a Workload exist for my Pod group?
+
+Before creating a Workload object, Kueue expects all the Pods for the group to be created.
+The Pods should all have the same value for the label `kueue.x-k8s.io/pod-group-name` and
+the number of Pods should be equal to the value of the annotation `kueue.x-k8s.io/pod-group-total-count`.
+
+You can run the following command to identify whether Kueue has or has not created a Workload
+for the Pod:
+
+```bash
+kubectl describe pod my-pod -n my-namespace
+```
+
+If Kueue didn't create the Workload object, you will see an output similar to the following:
+
+```
+...
+Events:
+  Type     Reason              Age   From                  Message
+  ----     ------              ----  ----                  -------
+  Warning  ErrWorkloadCompose  14s   pod-kueue-controller  'my-pod-group' group has fewer runnable pods than expected
+```
+
+Note that the above event might show up for the first Pod that Kueue observes, and it will remain
+even if Kueue successfully creates the Workload for the Pod group later.
+
+Once Kueue observes all the Pods for the group, you will see an output similar to the following:
+
+```
+...
+Events:
+  Type     Reason              Age   From                  Message
+  ----     ------              ----  ----                  -------
+  Normal   CreatedWorkload     14s   pod-kueue-controller  Created Workload: my-namespace/my-pod-group
+```
+
+## Why did my Pod disappear?
+
+When you enable [preemption](/docs/concepts/cluster_queue/#preemption), Kueue might preempt Pods
+to accomodate higher priority jobs or reclaim quota. Preemption is implemented via `DELETE` calls,
+the standard way of terminating a Pod in Kubernetes.
+
+When using single Pods, Kubernetes will delete Workload object along with the Pod, as there is
```

Comment:
Can we add a sentence on what happens with workload when using Pod groups?

Reply:
It's mostly the next question that covers it, but I added a note.